src/util.js

```diff
@@ -23,11 +23,37 @@ export const align = props => {
   return null
 }
 
+export const center = props => {
+  if (props.center || (props.centerX && props.centerY)) {
+    return {
+      top: '50%',
+      bottom: 'auto',
```

Comment:
The `auto` values probably shouldn't be returned here, as they'd wipe out styles if you did `<Absolute centerX bottom />` right?